the-collab-lab/tcl-28-smart-shopping-list
# New firebase config

## Description

Updates the firebase configuration because the previous firebase credentials expired. New credentials come from a new project on Firestore. See the link to the new firestore [here](https://console.firebase.google.com/project/lettuce-know/overview). If you can't view the firestore, then try logging in with the google credentials I put in Slack, and then try again.

The files that were updated:
```
firebase.json
```

The files that were added to the root directory:
``` 
.firebaserc
firestore.indexes.json // this file includes the index we need to query the database
firestore.rules
```

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓  | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

I'm not 100% sure how to test this without merging to the main branch, but if this works, then there should be a preview URL for the app when this PR is created. And if _that_ works, then I think we're okay.

So to test, `git checkout new-firebaseConfig`, maybe `npm install`?, then try joining an existing list with this token: `hit bale polka`. Since this is a new firestore database, we're starting from scratch with the data. You can also try the preview URL and test that.
